backend/db/utils.py

```diff
@@ -142,7 +142,7 @@ def refresh_derived_tables_exec(
      entry will appear further down in the list."""
     temp_table_suffix = '_new'
     ddl_modules_queue = derived_tables_queue
-    views = [x for x in list_views() if x in ddl_modules_queue]
+    views = [x for x in list_views(schema=schema) if x in ddl_modules_queue]
```

Comment:
Bug fix: Refreshes on `test_n3c` could error out because `list_views()` was not detecting views.

The reason that view `concept_set_member_plus` exists in `test_n3c` but not `n3c` is because it still exists in the DDL, which is no longer re-run for `n3c`, I believe, whereas `test_n3c` gets regenerated each time the tests run.